tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -788,7 +788,6 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
 
                         m(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
                             status should be(OK)
-                            header("content-type").get.toString shouldBe "content-type: application/json"
```

Comment:
Ok - if we have other tests that assert the content type header, do we need the additional integration test?